tools/check-source.sh

```diff
@@ -73,6 +73,10 @@ grep -ne '^\\\(constraints\|mandates\|expects\|effects\|sync\|ensures\|returns\|
 # Fixup: sed 's/^\\\(constraints\|mandates\|expects\|effects\|sync\|ensures\|returns\|throws\|complexity\|remarks\|errors\)\s*\(.\)/\\\1\n\2/'
 # Fixup: sed 's/^\\ //'
 
+# Order of library elements.
+../tools/element-order.awk *.tex |
+    fail '' || failed=1
```

Comment:
Do you not have a message here because the tool already prints its own message? Might be nice to have a generic message here regardless (if for nothing but documentation of the script itself), e.g. "incorrect ordering of library elements".

Reply:
Fixed + force-pushed.